brainiak/reprsimil/brsa.py

```diff
@@ -284,12 +270,12 @@ class BRSA(BaseEstimator, TransformerMixin):
         Note that nuisance regressor is not required from user. If it is
         not provided, DC components for each run will be included as nuisance
         regressor regardless of the auto_nuisance parameter.
-    n_nureg: int or string. Default: 'opt'
+    n_nureg: int or None. Default: None
```

Comment:
I would write:
`n_nureg: int`

This is also what the documentation I linked says ("An optional argument with a default needn’t use the Optional qualifier on its type annotation (although it is inferred if the default is None).") And also what you have already for `inten_smooth_range` and `space_smooth_range`.